src/__experimental__/components/input/input.style.js

```diff
@@ -29,4 +33,4 @@ StyledInput.propTypes = {
   disabled: PropTypes.bool
 };
 
-export default StyledInput;
+export { StyledInput as default, inputStyle };
```

Comment:
if this style is reused, then I think it should be moved to a separate file

Reply:
I think it is OK but I can separate it to new file named `form-field.style.js` in input directory.
What do you think? @andrewjtait do you have an opinion for it?

Reply:
I think you should follow the recommendation here: https://www.styled-components.com/docs/basics#extending-styles

Reply:
updated, thanks